stm32-data-gen/src/rcc.rs

```diff
@@ -373,6 +373,11 @@ impl PeripheralToClock {
             return clocks.get("ADC");
         }
 
+        // Absolute fallback, match against the clocks for just the first ADC
```

Comment:
could you make this a bit more restrictive, like `peri_name == "ADC"`? Otherwise it's a bit risky, it could cause silent failures like e.g. `ADC3` to fall back to `ADC1` clock bit.

Reply:
Done in the latest commit